src/actions/write/evolveState.ts

```diff
@@ -15,20 +15,25 @@ export const evolveState = async (
     throw new ContractError(NON_CONTRACT_OWNER_MESSAGE);
   }
 
-  // Modify contract name and ticker
-  state.name = 'Test IO';
-  state.ticker = 'tIO';
+  // An amount to airdrop to gateway testnet operators
+  const airdrop = 1500;
 
-  // Modify all balances
-  const perUserBalance = 5_000;
-  state.balances[owner] = 1_000_000_000; //give the token supply to the owner for distribution
-  const balances = state.balances;
-  Object.keys(balances).forEach((address) => {
-    if (address !== owner) {
-      state.balances[address] = perUserBalance;
-      state.balances[owner] -= perUserBalance;
-    }
-  });
+  // Set each gateway to have an empty array of vaults
+  for (const address in state.gateways) {
```

Comment:
we don't want to airdrop the owner - do an if check and just `continue` on it

Reply:
FWIW you could use the previous implementation and just swap out the airdrop in place of perUserBalance

Reply:
Put in a check to only airdrop if address !== owner